### PR TITLE
Improve install-deps

### PR DESCRIPTION
**Pull request by ermshiperete**
We want to install any Ubuntu packages in non-interactive mode.

---
This change is [Reviewable](https://reviewable.io/reviews/bloombooks/bloomdesktop/1258)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -97,3 +97,4 @@ Andika-R.ttf
 Bloom/
 .vs/config/applicationhost.config
 build/Bloom*.nupkg
+src/BloomBrowserUI/filesToFix.lst
```
Comment (hatton) [inline .gitignore]: What creates this?

Comment (ermshiperete) [inline .gitignore]: Building on Linux from the command line. Gets created by `src/BloomBrowserUI/fixNodejsForLinux.sh`.

**Comment by StephenMcConnel** [comment]
Generally looks okay, but one question that I'd like answered.

---

Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[.gitignore, line 100 at r1](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1258#-KUXHORe-1-WATpE-xLP-r1-100:-KUZFm6RSqTgOiLYu6nR:b-p6f3ao) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/6e306ca48f3d5544d89b4c6baa1f632e9453fcb6/.gitignore#L100)):*
<details><summary><i>Previously, ermshiperete (Eberhard Beilharz) wrote…</i></summary>
> Building on Linux from the command line. Gets created by `src/BloomBrowserUI/fixNodejsForLinux.sh`.
</details>
that's correct.  I'm not sure that hack is needed for newer versions of nodejs on linux, but haven't felt compelled to check that.

---

*[build/install-deps, line 33 at r1](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1258#-KUZGPoFQ9qzRfkPObJv:-KUZGPoFQ9qzRfkPObJw:btt7fs5) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/6e306ca48f3d5544d89b4c6baa1f632e9453fcb6/build/install-deps#L33)):*
> ```
> 
> export DEBIAN_FRONTEND=noninteractive
> sudo apt-get update
> ```

does sudo pick up the current environment?  One site on the internet showed "sudo DEBIAN_FRONTEND=noninteractive apt-get -y install whatever", which would place this environment variable setting elsewhere.  (does apt-get update ever pause to ask the user anything?)  just a question.  if you've tested this and it works, I'll take your word for it.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1258#-:-KUZHRx7lcRpSKZZGOZj:b9lyrdq)*

**Comment by hatton** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[.gitignore, line 100 at r1](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1258#-KUXHORe-1-WATpE-xLP-r1-100:-KUZSCkHpmgGuQsIVQmA:bpsvyu9) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/6e306ca48f3d5544d89b4c6baa1f632e9453fcb6/.gitignore#L100)):*
<details><summary><i>Previously, StephenMcConnel (Steve McConnel) wrote…</i></summary>
> that's correct.  I'm not sure that hack is needed for newer versions of nodejs on linux, but haven't felt compelled to check that.
</details>
COuld this .sh instead go in build/  ? And maybe pipe its output to output/? (sorry, not really what E is doing here, just first time it has come to my attention.).

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1258)*

**Comment by ermshiperete** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[.gitignore, line 100 at r1](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1258#-KUXHORe-1-WATpE-xLP-r1-100:-KUac29wAtYU_WoOuiG5:b-5aml3) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/6e306ca48f3d5544d89b4c6baa1f632e9453fcb6/.gitignore#L100)):*
<details><summary><i>Previously, hatton (John Hatton) wrote…</i></summary>
> COuld this .sh instead go in build/  ? And maybe pipe its output to output/? (sorry, not really what E is doing here, just first time it has come to my attention.).
</details>
I'm with Steve that I'm not sure that the hack is needed nowadays - seemed to work without, but I didn't look what this script actually does so I can't tell. If we still need it, moving it into build/ would be a good idea, but that's a different task :smiley:

---

*[build/install-deps, line 33 at r1](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1258#-KUZGPoFQ9qzRfkPObJv:-KUabcbjVqFNswAmyCrQ:b94163k) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/6e306ca48f3d5544d89b4c6baa1f632e9453fcb6/build/install-deps#L33)):*
<details><summary><i>Previously, StephenMcConnel (Steve McConnel) wrote…</i></summary>
> does sudo pick up the current environment?  One site on the internet showed "sudo DEBIAN_FRONTEND=noninteractive apt-get -y install whatever", which would place this environment variable setting elsewhere.  (does apt-get update ever pause to ask the user anything?)  just a question.  if you've tested this and it works, I'll take your word for it.
</details>
I've been in that trap before! Yes, you're right, it needs to go in the sudo line. 

I copied this from a script that gets run as root, so it works there, but wouldn't work here. Thanks!

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1258)*

**Pull request merged**